diff -Nru aggdraw-1.1-20051010.orig/setup.py aggdraw-1.1-20051010/setup.py
--- aggdraw-1.1-20051010.orig/setup.py	2005-10-15 14:55:00.000000000 +0200
+++ aggdraw-1.1-20051010/setup.py	2007-10-14 16:08:56.745390000 +0200
@@ -18,7 +18,7 @@
 VERSION = "1.1-20051010"
 
 # tweak as necessary
-FREETYPE_ROOT = "../../kits/freetype-2.1.10"
+FREETYPE_ROOT = "/usr"
 
 if not os.path.isdir(FREETYPE_ROOT):
     print "===", "freetype support disabled"
